linux/user-and-file-management/linux-intro/linux-common-terminology.md

```diff
@@ -0,0 +1,60 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+
+---
+
+# Common Terminology
+
+---
+## Content
+
+
+Let's go over a few terms that you'll see a lot in this course.
+
+A **command** is just a piece of text that you type into the **terminal**. Think of this as an instruction that you give to your computer.
+
+Sometimes you might type some commands which are restricted. That's where the **root user** comes in. It's the unique admin user, with superpowers. Unsurprisingly, it's also called the **superuser**.
+
+The **package manager** removes the tedious process of installing or uninstalling apps. Think of a **package** as being an app.
+
+Remember the terminal? Well, this is also referred to as the **shell**. 
```

Comment:
The terminal is mentioned a few sentences above so "remember" seems unnecessary?

Also, a shell isn't really the same as the terminal. A shell is a program running in the terminal.

Reply:
Maybe this part about the shell can be:

```suggestion
The main part of the terminal that reads the commands and executes them is called a **shell**
```